Makefile.PL

```diff
@@ -42,8 +42,8 @@ EOT
 
 
 if ($test_for_tk) {
-    open TCLSH, "$tclsh test-for-tk.tcl|";
-    my $res = join '', <TCLSH>;
+    open my $tk_test, "$tclsh test-for-tk.tcl|";
+    my $res = join '', <$tk_test>;
```

Comment:
I'm not requesting changes; I've just wondered if it would be alright to use `my $res = qx/$tclsh test-for-tk.tcl/` instead for simplicity.

Reply:
That's  good idea; i wanted to focus the first patch on just the barewords to keep it tight and easy to merge.  The plan would be to then update the `open` statements to use the three-arg form where possible.  In this case, the three-arg form would be unnecessary, since one could replace it with a `qx{}`, so thanks for mentioning this :-)